api/services/DynamoDBDocumentHelper.js

```diff
@@ -0,0 +1,54 @@
+const AWS = require('aws-sdk');
+const _ = require('underscore');
+const { logger } = require('../../winston');
+
+const logError = fn => fn.promise()
+  .catch((err) => {
+    logger.error(err);
+    throw err;
+  });
+
+class DynamoDBDocumentHelper {
+  constructor(credentials) {
+    this.docClient = new AWS.DynamoDB.DocumentClient({
+      accessKeyId: credentials.accessKeyId,
+      secretAccessKey: credentials.secretAccessKey,
+      region: credentials.region,
+    });
+  }
+
+  put(TableName, Item) {
```

Comment:
Does not need to be resolved now, and I assume you did this for conciseness, but I think the function arguments should be camelcase

Reply:
yeah .. .it can be changes ... this titlezed variables is the way the objects are defined in all of the AWS documentation I've seen.  we could create untitlized variables with the same name for uniformity